Dockerfile

```diff
@@ -1,19 +1,17 @@
-FROM quay.io/ukhomeofficedigital/openjdk11:v11.0.5_10
+FROM quay.io/ukhomeofficedigital/alpine:3.10
 
 ENV USER user_hocs_workflow
 ENV USER_ID 1000
 ENV GROUP group_hocs_workflow
 ENV NAME hocs-workflow
 ENV JAR_PATH build/libs
 
-RUN yum update -y glibc && \
-    yum update -y nss && \
-    yum update -y bind-license
+RUN apk add openjdk11-jre
```

Comment:
Can this be a specific version v11.0.5_10 ?

Reply:
It is specifically 11.0.4_p4-r1 despite the unclear package name, see https://pkgs.alpinelinux.org/package/v3.10/community/s390x/openjdk11-jdk

Reply:
is there a later alpine version that matchs  v11.0.5_10? 
Just thinking about maintaining versions.

Reply:
Moved to alpine 3.13. which gives us Java 11.0.9_11. Nothing in the release notes stand out to me as an obvious problem

Reply:
Sounds good.